initials.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
```

Comment:
Great job! Your code compiled and ran - the only thing I'd add is a return 0; and a comment at the top of your code with your name and function of your program.